.travis.yml

```diff
@@ -1,22 +1,28 @@
+language: python
 sudo: false
 cache: pip
 
+python:
+  - "2.7"
+
 addons:
   apt:
     packages:
       - expect-dev  # provides unbuffer utility
       - python-lxml # because pip installation is slow
 
-language: python
-
-python:
-  - "2.7"
-
 env:
-  - VERSION="8.0" LINT_CHECK="1"
-  - VERSION="8.0" ODOO_REPO="odoo/odoo" LINT_CHECK="0"
-  - VERSION="8.0" ODOO_REPO="OCA/OCB" LINT_CHECK="0"
-  - VERSION="8.0" UNIT_TEST="1" LINT_CHECK="0"
+  global:
+  - VERSION="8.0" TESTS="0" LINT_CHECK="0" TRANSIFEX="0"
+  - TRANSIFEX_USER='transbot@odoo-community.org'
+  - secure: J6Zr34aVnjXFRwIamLmnlsN42t8ki2syAvQ26xJ+uOuJUkDLjSDnvrkFtss/Wn1watIERSrw/T8TrAZ0GxhUQ7B8jq7EZFmewobhyBpQVBR58S2clcGupYXgLUk9GwYfbxb7uAgIM8YOlnIteE69gMXJubzBx5pxEHOayUPbc/s= 
```

Comment:
I have 'OCA/server-tools': 'Z06mZCN+Hm3myqHSOZpOOk1pd4oq1epAWZv6m9OX2bTNHbhyOVOGK6JWWsnDm/3DUCN1ZeLtSGOl9bvQfMa8ahQHA80MkLL16YlTvQV59Lh+L2gAYmxX+ogJCJgeQSVAXlGLscgkADCu/HzDlmatrDeROMtULn5i23j2qcyUNyM=',

On https://github.com/OCA/maintainer-quality-tools/issues/194

We definitively needs to do that with a script for this I guess.